src/main/java/de/tracetronic/jenkins/plugins/ecutest/report/atx/pipeline/ATXServer.java

```diff
@@ -85,6 +85,20 @@ public void publish(final boolean allowMissing, final boolean runOnFailed,
         script.invokeMethod("publishATX", stepVariables);
     }
 
+    /**
+     * Publishes ATX reports with given archiving settings as named arguments map.
+     *
+     * @param settings the settings map
+     */
+    @Whitelisted
+    public void publish(final Map<String, Object> settings) {
+        final Map<String, Object> stepVariables = Maps.newLinkedHashMap();
+        stepVariables.put("atxName", installation.getName());
+        stepVariables.put("atxInstallation", installation);
+        stepVariables.putAll(settings);
```

Comment:
Should not we add also other named parameters: `allowMissing`, `runOnFailed`, `archiving`, `keepAll`